index.js

```diff
@@ -6,6 +6,7 @@ class SubSignalhub {
   constructor(instance, prefix) {
     this._instance = instance
     this._prefix = prefix
+    this.app = instance.app + this._prefix;
```

Comment:
That's a good point, I forgot to proxy that field. Is there anything else that's missing from the signalhub API?

Reply:
I'm checking the mafintosh/webrtc-swarm and I think is not using anything else. So, we are good.